<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fusion3DClass</class>
 <widget class="QMainWindow" name="Fusion3DClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3d_reconstruction</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnImagePath">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>img_set_path</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>305</number>
    </property>
    <property name="autoRepeatInterval">
     <number>95</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOutScene">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>scene_path</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>305</number>
    </property>
    <property name="autoRepeatInterval">
     <number>95</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagePath">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>22</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_outpath">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_sub">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>701</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_total">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>701</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="img_path_msg_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>input imageset path.</string>
    </property>
   </widget>
   <widget class="QLabel" name="scene_path_msg_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>input out path.!</string>
    </property>
   </widget>
   <widget class="QLabel" name="ply_path_error_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>there is no model. please check it..</string>
    </property>
   </widget>
   <widget class="QLabel" name="sample_len_err_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>please set sample length.</string>
    </property>
   </widget>
   <widget class="QLabel" name="recon_done_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>reconstruction is done.</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_from_recon_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="midLineWidth">
     <number>4</number>
    </property>
    <property name="text">
     <string>label</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>569</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="akaze_check">
       <property name="text">
        <string>akaze</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="sift_check">
       <property name="text">
        <string>sift</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="akaze_sift_check">
       <property name="text">
        <string>all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mvs_check">
       <property name="text">
        <string>mvs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="smvs_check">
       <property name="text">
        <string>smvs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="plane_remove_check">
       <property name="text">
        <string>remove_floor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>plane_thick</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="plane_tolerance_edt"/>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="use_shade_check">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>shading</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>212</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="default_setting_check">
       <property name="text">
        <string>default_setting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="seperate_recon_check">
       <property name="text">
        <string>step_recon</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>549</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>image_scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="img_size_spin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>input_scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="input_scale_spin">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>out_scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="out_scale_spin">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>sample_lenth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sample_len_edt"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>203</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnExecute">
       <property name="text">
        <string>entire_recon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openModelBtn">
       <property name="text">
        <string>scaling_model</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>495</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="scene_btn">
       <property name="text">
        <string>make_scene</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sfm_btn">
       <property name="text">
        <string>sfm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dm_btn">
       <property name="text">
        <string>depth_recon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fss_btn">
       <property name="text">
        <string>surface_recon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tex_btn">
       <property name="text">
        <string>tex_recon</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="tex_scale_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>scaling_tex_mesh</string>
    </property>
   </widget>
   <widget class="QLabel" name="scale_err_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>no any scale. please check it..</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Fusion3D.qrc"/>
 </resources>
 <connections/>
</ui>
